src/components/Transaction/Transactions.js

```diff
@@ -9,11 +10,20 @@ export default function Transactions() {
 	const db = useStore(state => state.db);
 	const {accountID} = useParams();
 	const entries = db.find(account => account.id === accountID).entries;
+	const navigate = useNavigate();
 
 	return (
 		<>
 			{entries.map(({id, date, change, note}) => (
-				<Transaction key={id} date={date} change={change} note={note} />
+				<Transaction
+					key={id}
+					date={date}
+					change={change}
+					note={note}
+					onEdit={() => {
+						navigate('/transaction/edit/' + accountID + '/' + id);
```

Comment:
I usually prefer template strings but both are valid solutions :)
```suggestion
						navigate(`/transaction/edit/${accountID}/${id}`);
```